libcxx/utils/libcxx/sym_check/util.py

```diff
@@ -123,6 +123,42 @@ def adjust_mangled_name(name):
     return name[1:]
```

Comment:
Not attached to this line: let's re-generate the ABI lists on Apple as part of this patch, unless there's a reason you opted not to do that.

Reply:
Great. I will have that in the next version!